src/Microsoft.Framework.PackageManager/Building/BuildManager.cs

```diff
@@ -198,17 +198,20 @@ public bool Build()
                 }
             }
 
-            if (!ScriptExecutor.Execute(project, "postbuild", GetScriptVariable))
+            if (success)
             {
-                LogError(ScriptExecutor.ErrorMessage);
-                return false;
-            }
+                if (!ScriptExecutor.Execute(project, "postbuild", GetScriptVariable))
+                {
+                    WriteError(ScriptExecutor.ErrorMessage);
+                    success = false;
+                }
 
-            if (_buildOptions.GeneratePackages &&
-                !ScriptExecutor.Execute(project, "postpack", GetScriptVariable))
-            {
-                LogError(ScriptExecutor.ErrorMessage);
-                return false;
+                if (_buildOptions.GeneratePackages &&
+                    !ScriptExecutor.Execute(project, "postpack", GetScriptVariable))
```

Comment:
previous code would have bailed out without executing `postpack` scripts if `postbuild` scripts failed (see the removed `return` at old line 204).  now runs those scripts any time earlier steps (i.e. the build) succeed.
suggest this `if` should include `success &&` in the condition or (worse) add `_buildOptions.GeneratePackages = false;` above line 207.